test/unit/range/views/view_minimiser_test.cpp

```diff
@@ -23,14 +25,27 @@
 using seqan3::operator""_dna4;
 using seqan3::operator""_shape;
 using result_t = std::vector<size_t>;
-using iterator_type = std::ranges::iterator_t< decltype(std::declval<seqan3::dna4_vector&>()
-                                                        | seqan3::views::kmer_hash(seqan3::ungapped{4})
-                                                        | seqan3::views::minimiser(5))>;
 
-static constexpr auto kmer_view = seqan3::views::kmer_hash(seqan3::ungapped{4});
-static constexpr auto gapped_kmer_view = seqan3::views::kmer_hash(0b1001_shape);
-static constexpr auto minimiser_view = seqan3::views::minimiser(5);
-static constexpr auto minimiser_view2 = seqan3::views::minimiser(1); // kmer_size == window_size, should throw
+inline static constexpr auto kmer_view = seqan3::views::kmer_hash(seqan3::ungapped{4});
+inline static constexpr auto rev_kmer_view = seqan3::views::complement | std::views::reverse
+                                                                       | kmer_view
+                                                                       | std::views::reverse;
+inline static constexpr auto gapped_kmer_view = seqan3::views::kmer_hash(0b1001_shape);
+inline static constexpr auto rev_gapped_kmer_view = seqan3::views::complement | std::views::reverse
+                                                                              | seqan3::views::kmer_hash(0b1001_shape)
+                                                                              | std::views::reverse;
+inline static constexpr auto minimiser_view1 = seqan3::views::minimiser(1); // kmer_size == window_size
+inline static constexpr auto minimiser_no_rev_view = seqan3::views::minimiser(5);
+
+using iterator_type = std::ranges::iterator_t< decltype(std::declval<seqan3::dna4_vector&>()
+                                               | kmer_view
+                                               | minimiser_no_rev_view)>;
+using two_ranges_iterator_type = std::ranges::iterator_t< decltype(seqan3::detail::minimiser_view{
+                                                          std::declval<seqan3::dna4_vector&>()
+                                                          | kmer_view,
+                                                          std::declval<seqan3::dna4_vector&>()
+                                                          | rev_kmer_view,
```

Comment:
which one of this is not const-iterable? I think both should, right? I really like to have the iterator test work on the const iterable range to check that everything indeed works. Because const-iterability concept does not test if the const iterator can be constructed from the non-const iterator etc.. So the iterator test template should run with the const_iterable enabled.

Reply:
@rrahn Can I do that in a follow-up issue? I really would like to use the minimizers.

Reply:
@rrahn rev_kmer_view makes it not const_iterable because `text | seqan3::views::kmer_hash(seqan3::ungapped{4}) | std::views::reverse` is not const_iterable. If you want I can create an issue about that

Reply:
@rrahn *polite ping*

Reply:
Ok, please make a backlog item for it. 
Because, neither kmer_hash nor reverse should remove const-iterability, shouldn't it? Or does the kmer-hash view stores a state?

Reply:
Backlog entry: https://github.com/seqan/product_backlog/issues/161